### PR TITLE
Updating csi plugin to support multi-arch builds

### PR DESCRIPTION
**Pull request by jaidevmane**
Updated csi plugin to use the new konveyor/builder image to support multi-arch builds

### CODE REVIEW

**Review by sseago — COMMENTED** [review]
Same question here as I had on the openshift-velero-plugins one -- is the goal just to swap out the builder image, or do we want to use konveyor/builder as the *running* container image as well, which is what this PR does?

**Review by dymurray — CHANGES_REQUESTED** [review]
Please change only the builder image

File: Dockerfile.ubi [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM quay.io/konveyor/builder as builder
+ENV GOPATH=$APP_ROOT
+COPY . $APP_ROOT/src/velero-plugin-for-csi
+WORKDIR $APP_ROOT/src/velero-plugin-for-csi
+RUN CGO_ENABLED=0 GOOS=linux go build -v -o $APP_ROOT/bin/velero-plugin-for-csi -mod=mod .
+
+
+FROM quay.io/konveyor/builder
```
Comment (dymurray) [inline Dockerfile.ubi:8]: This should still be using `registry.access.redhat.com/ubi8-minimal`

**Comment by jmontleon** [comment]
I'll be creating a PR shortly to address this and enable multiarch builds.

**Pull request closed without merging**